### ISSUE TITLE
MDTraj 1.4 Timeline

### ISSUE THREAD

**Issue by kyleabeauchamp** [mdtraj/mdtraj#830]
Given that Ipython 3.0 is now standard and the current MDTraj notebook stuff requires IPython 3.0, it might make sense to do a release at some point.  We should see what other immediate bugs need fixing first.  See also #829

@davidmobley I actually doubt we can get a release in time for your class, so the easiest thing to do might be to have them force install IPython 2.0 using something like `conda install ipython==2.4.1` or something

**Comment by kyleabeauchamp** [comment]
Meant to say @davidlmobley

**Comment by davidlmobley** [comment]
@kyleabeauchamp - This is naive, but if they have both ipython versions installed, how do they pick which one they use? It looks like I could handle it using environments but then I'll have to teach them how to do that, I think... Is there an easier way?

**Comment by kyleabeauchamp** [comment]
I think that command will downgrade:


```
kyleb@kyleb-DX4870:~$ conda install ipython==2.4.1
Fetching package metadata: ....
Solving package specifications: .
Package plan for installation in environment /home/kyleb/opt:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ipython-2.4.1              |           py27_0         2.8 MB

The following packages will be DOWNGRADED:

    ipython: 3.0.0-py27_0 --> 2.4.1-py27_0

Proceed ([y]/n)? 

```

**Comment by davidlmobley** [comment]
So that will break all their existing notebooks until they, er, upgrade again. Hmm. I guess I'll just have to have them install the master branch from git, though it will cause headaches...

**Comment by kyleabeauchamp** [comment]
Let's see what Robert says regarding the release.  I'll be at a department retreat tomorrow, so I'll be slow to respond.

**Comment by swails** [comment]
@davidlmobley -- try installing `mdtraj-dev` from conda (it looks like there's a package like that, right?: https://conda.binstar.org/omnia)

Does that work?

**Comment by swails** [comment]
I *think* that should be the latest snapshot that Travis built (@rmcgibbo can correct me if I'm wrong)

**Comment by davidlmobley** [comment]
Yes, that does it. Thanks!

On Wed, May 20, 2015 at 4:44 PM, Jason Swails <notifications@github.com>
wrote:

> I *think* that should be the latest snapshot that Travis built (@rmcgibbo
> <https://github.com/rmcgibbo> can correct me if I'm wrong)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mdtraj/mdtraj/issues/830#issuecomment-104073009>.
>



-- 
David Mobley
dmobley@gmail.com
949-385-2436

**Comment by rmcgibbo** [comment]
I'm in Korea for the next ten days (just landed), so I probably can't do a
release before the first week of June.

I think @mpharrigan is also on vacation. If someone else wants to do it
though, that would be great. The more the merrier.

**Comment by kyleabeauchamp** [comment]
I too am traveling for several days, so I think the release will wait.

**Comment by rmcgibbo** [comment]
I'm back from vacation now? Should we do a release. Here's the list of PRs merged since 1.3.0. It seems like a pretty reasonable number. I can also merge #837 once the windows tests pass.

```
$ gh-util list-prs mdtraj/mdtraj
```
Listing merged PRs since: MDTraj-1.3.0

PR #836: BUGFIX: Fix gro file unitcell handling
PR #835: Make virtual site a new element
PR #825: ENH: Recognize .ncrst and .rst7 for trajectory save extensions
PR #3: Fix PR on windows
PR #826: typo in code example
PR #823: MAINT: Pin sphinx version to avoid regression.
PR #818: ENH: Add full list of van der Waals radii
PR #817: Fix pytables deprecation error
PR #816: Fix update-versions.py script, porting msmbuilder#550
PR #811: MAINT: Increase minimum required python version to 2.7
PR #813: BUG: Fix test_unitcell from my refactoring of test_trajectory
PR #810: ENH: Support .gz and .bz2
PR #804: Make vmd-readable .gro
PR #808: Add .ncdf as a recognized suffix for the NetCDF loader
PR #802: DOC: Comment about atom.serial
PR #799: ENH: Add save_gro
PR #795: Import `load_xyz` in top level namespace
PR #791: IPython viewer in docs
PR #788: load_hoomdxml and tests for load_hoomdxml
PR #781: Optional `n_bins` parameter for compute_rdf
PR #789: fix IPython 3.0 compatibility issues
PR #784: BUG:Fix garbage in DCD out filename
PR #782: Download 4OH9 so that test_dihedral doesn't contact the outside world
PR #779: Update index.rst
PR #771: Updating docs for nematic order
PR #773: Support .pdb.gz files and make loading multiple pdb files more efficient
PR #778: ENH: Support Mol2 status bits
PR #776: Fix compute_rdf r_range argument
PR #775: Cleanup in from_dataframe
PR #759: [ENH] Compute nematic order parameter
PR #770: Add chainid to DSL
PR #760: Option to pre-compute pairs for RDF
PR #764: Fix bug in xyz writer and add appropriate unit test
PR #761: n_chains convenience function for traj
PR #762: Update load_functions.rst
PR #763: Update README.md
PR #758: Update README.md
PR #755: Update whatsnew.rst
PR #754: Update building_docs.rst
PR #757: test_rmsd_memmap: use tempfile to make directory for tmp files rather th...
PR #752: MAINT: Update notebook examples to py3
PR #750: Update CentOS 5 instructions
PR #749: Compile on vsp-compute needs g++44 too
PR #747: [ENH] Reader for .xyz files
PR #746: Overwrite not being passed to HDF5 and .lammpstrj and some small stuff (doc strings, pep8)
PR #745: Fix some docstrings
PR #748: Add compilation fixes for extension modules
PR #744: [ENH] Improved efficiency of generating RDF pairs
PR #743: Switch sse_swizzle to the MIT license after email with Imran
PR #742: Release 1.3.0

**Comment by rmcgibbo** [comment]
I did the release. Binaries should be up shortly for conda.
